include/deal.II/fe/mapping_fe_field.h

```diff
@@ -62,50 +62,91 @@ DEAL_II_NAMESPACE_OPEN
  *    dhq.distribute_dofs(fesystem);
  *    Vector<double> eulerq(dhq.n_dofs());
  *    const ComponentMask mask(spacedim, true);
- *    MappingFE<dim,spacedim> map(eulerq, dhq, mask);
+ *    MappingFEField<dim,spacedim> map(eulerq, dhq, mask);
  *    map.update_euler_vector_using_triangulation(eulerq);
```

Comment:
> Morally, that function belongs to |VectorTools|, not to a member function of
> MappingFEField.

I see, indeed. Maybe that's where the function should then be. It would allow 
calling it without an object of type `MappingFE`.